xbmc/addons/kodi-addon-dev-kit/include/kodi/libXBMC_addon.h

```diff
@@ -224,7 +201,7 @@ namespace ADDON
      * @param type The message type.
      * @param format The format of the message to pass to display in XBMC.
      */
-    void QueueNotification(const queue_msg_t type, const char *format, ... )
+    void QueueNotification(const QueueMsg type, const char *format, ... )
```

Comment:
Can or should this be a reference?

Reply:
Also, `const char* format, ...`

Left align `*`

Reply:
Ah thanks, has not thought about :smile: